trieste/acquisition/optimizer.py

```diff
@@ -162,16 +162,31 @@ def _objective() -> TensorType:
             return -target_func(bijector.forward(variable[:, None, :]))  # [1]
 
         chosen_points = tf.ones([0, len(space.lower)], dtype=trial_search_space.dtype)  # [0, D]
-        for i in tf.range(num_restarts):  # restart optimization
+        for i in tf.range(num_restarts):  # perfom optimization from chosen starting points
```

Comment:
possible confusion: if restart=1 you would run 2 optimisations? change to num_bfgs_runs for clarity?